applications/models.py

```diff
@@ -71,6 +71,21 @@ def __str__(self):
 
 
 class Organisation(models.Model):
+    COMPANY = 1
```

Comment:
Everywhere else in the system we currently use strings as identifiers for choices. For example application status:

```
    DRAFT = "draft"
    IN_REVIEW = "in_review"
    REVIEW_DONE = "review_done"
    ALLOCATING = "allocating"
    ALLOCATED = "allocated"
    VALIDATED = "validated"
    DECLINED = "declined"
    CANCELLED = "cancelled"
    HANDLED = "handled"

    STATUS_CHOICES = (
        (DRAFT, _("Draft")),
        (IN_REVIEW, _("In review")),
        (REVIEW_DONE, _("Review done")),
        (ALLOCATING, _("Allocating")),
        (ALLOCATED, _("Allocated")),
        (VALIDATED, _("Validated")),
        (DECLINED, _("Declined")),
        (CANCELLED, _("Cancelled")),
        (HANDLED, _("Handled")),
    )
```

I think it kinda makes API content more readable because you can't know which company type each integer if without reading the code.